src/packaging/publish_release.py

```diff
@@ -67,13 +67,17 @@
 
 if not releaseExists:
     print('Creating release "%s" with tag "%s" on commit "%s"' % (title, tag, options.commit))
+    if tag:
+        draft = True
+    else:
+        draft = False
```

Comment:
You can replace these line by (but if you think it is more understandable like this it's fine for me too):
```
draft = True if tag else False
```